middle_end/flambda/unboxing/unboxing_types.mli

```diff
@@ -0,0 +1,118 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                       Pierre Chambart, OCamlPro                        *)
```

Comment:
It's minor but I guess I should also be mentioned in these headers ? (If only to be blamed later when things break, ^^)

Reply:
I'm planning on rewriting all the headers actually (possibly even removing them in favour of a single file at toplevel), so we can probably leave this for now (but push a changeset if it particularly bothers you, of course)!

Reply:
No worries then, ^^